### PR TITLE
config.md: add information on `-c` option

### PR DESCRIPTION
**Pull request by deepend-tildeclub**
added -c option to the how to run the bot.   Without it I'm sure a ton of people had troubles with module loading and just gave up.

### CODE REVIEW

**Comment by examknow** [comment]
If the current configuration instructions are followed correctly, there is no need to pass the `-c` option as BitBot will look for the config file in `~/.bitbot/` by default.

**Comment by jesopo** [comment]
i'm sure a lot of people want to keep the config file in a non-default location?

**Comment by examknow** [comment]
Fair, however, I think it should be made more clear that this is optional and only necessary if the config file is located in a non-standard location.

**Comment by deepend-tildeclub** [comment]
> If the current configuration instructions are followed correctly, there is no need to pass the `-c` option as BitBot will look for the config file in `~/.bitbot/` by default.

Yeah totally misread that when I set it up.. So that is my fault for sure.

**Comment by examknow** [comment]
It may be worth having a section describing all of the optional arguments that can be passed to `bitbotd` although the argparser help already accomplishes this IMHO.

File: docs/help/config.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 * Move `docs/bot.conf.example` to `~/.bitbot/bot.conf` and fill in the config options you care about. Ones blank or removed will disable relevant functionality.
 * Run `./bitbotd -a` to add a server.
-* Run `./bitbotd` to start the bot.
+* Run `./bitbotd` to start the bot or './bitbotd -c /path/to/bot.conf' for non-standard config location.
```
Comment (examknow) [inline docs/help/config.md]:
```suggestion
* Run `./bitbotd` to start the bot or `./bitbotd -c /path/to/bot.conf` for non-standard config location (outside of `~/.bitbot`).
```

**Review by examknow — CHANGES_REQUESTED** [review]
one small formatting nitpick

**Review by examknow — APPROVED** [review]
If everybody's good with this, I'll merge later tonight. Thanks!

**Pull request merged**